apps/emqx/src/emqx_persistent_session_ds.erl

```diff
@@ -700,14 +697,6 @@ handle_timeout(
         Session0#{s := S, shared_sub_s := SharedSubS, stream_scheduler_s := SchedS}
     ),
     {ok, [], push_now(Session)};
-handle_timeout(_ClientInfo, ?TIMER_BUMP_LAST_ALIVE_AT, Session0 = #{s := S0}) ->
```

Comment:
I'm not sure we can remove this timer entirely. It's currently used to periodically dump session state to the DB; we should keep that.

Reply:
Yes, I am adding it back.